internal/perf/get_txs.go

```diff
@@ -0,0 +1,8 @@
+package perf
+
+import "github.com/hyperledger/firefly-perf-cli/internal/conf"
+
+func (pr *perfRunner) RunGetTransactions(id int) {
+	targeter := pr.getApiTargeter("GET", "transactions", "")
+	pr.runAttacker(targeter, id, conf.PerfCmdGetTransactions.String())
```

Comment:
Attacker seems aggressive 🙃 this is a perf test after all. Fine line between performance and stress though I suppose.

Reply:
Removed to only submit one request